library/options.js.php

```diff
@@ -101,10 +112,15 @@ function checkSkipConditions() {
 
     var trgelem1 = document.getElementById('label_id_' + target);
     var trgelem2 = document.getElementById('value_id_' + target);
+
     if (trgelem1 == null && trgelem2 == null) {
-      if (!cskerror) alert('<?php echo xls('Cannot find a skip target field for'); ?> "' + target + '"');
-      myerror = true;
-      continue;
+        var trgelem1 = document.getElementById('label_' + target);
+        var trgelem2 = document.getElementById('text_' + target);
+        if (trgelem1 == null && trgelem2 == null) {
+            if (!cskerror) alert('<?php echo xls('Cannot find a skip target field for'); ?> "' + target + '"');
+                myerror = true;
+                continue;
```

Comment:
The spacing in above two lines is confusion, since they are not part of the if (!cskerror conditional). should instead be:
``````
if (!cskerror) alert('<?php echo xls('Cannot find a skip target field for'); ?> "' + target + '"');
myerror = true;
continue;